src/main/java/mods/eln/Eln.java

```diff
@@ -2888,20 +2948,19 @@ private void registerMacerator(int id) {
             name = TR_NAME(Type.NONE, "200V Macerator");
 
             MaceratorDescriptor desc = new MaceratorDescriptor(name,
-                "maceratorb", MVU, 400,// double nominalU,double nominalP,
-                MVU * 1.25,// double maximalU,
-                new ThermalLoadInitializer(80, -100, 10, 100000.0),// thermal,
-                meduimVoltageCableDescriptor,// ElectricalCableDescriptor
-                // cable
+                "maceratorb", MVU, 1000,
```

Comment:
Also this. The change itself is fine, but it needs to be included in the descripotion.